src/changelogs/tags.py

```diff
@@ -0,0 +1,156 @@
+# Lists and maps regarding changelog tags
+
+
+class ChangelogTags:
+    def __init__(self, default_tag):
+        self.default_tag = default_tag
+        # Tags to register if they are found in the changelog line
+        # match by text
+        # never put lowercase tags here, as they are likely to be
+        # part of a longer string. Put them in self.tags_match_word instead.
+        # See the example for tag_remap for reasoning on shorter/longer strings
+        # Lowercase words are more likely to be a prefix/suffix of an uppercase word
+        # throwing a false positive match. Such as:
+        # "Return Fire changed from...", "urn" would be incorrectly tagged
+        # as such, it should only be matched if its a standalone word
+        self.match_text = [
+            'Trooper',
+            'Base Guardian',
+            'Base guardian',
+            'base guardian'
+            'Guardian',
+            'Walker',
+            'Patron',
+            'Weakened Patron',
+            'Weakened patron',
+            'Shrine',
+            'Mid-Boss',
+            'Midboss',
+            'MidBoss',
+            'Mid Boss',
+            'Mid boss',
+            'Rejuvenator',
+            'Creep',
+            'Neutral',
+            'Denizen',
+            'Golden Statue',
+            'Soul',
+            'Rope',
+            'Zipline',
+            'bounce pad',
+            'Bounce Pad',
+            'Bounce pad',
+        ]
+        # match by word
+        # add tags here instead of tags_match_text
+        # if they are a shorter string, or likely to be part of a longer string
+        # or if the lower case also needs to be matched
+        self.match_word = [
+            'creep',
+            'neutral',
+            'creeps',
+            'neutrals',
+            'Rejuv',  # so it doesnt get caught by Rejuvenating Aura
+            'Map',
+            'urn',
+            'Urn',
+            'urns',
+            'Urns',
+            'orb',
+            'orbs',
+            'Orb',
+            'Orbs',
+            'soul',
+            'souls',
+            'rope',
+            'ropes',
+            'zipline',
+            'ziplines',
+        ]
```

Comment:
Can we just have a check for lowercase. ie. `input_str.lowercase() === 'orbs'`? Similar with the other tag lists

Reply:
Could you elaborate on this idea? I just modified some of the guide phrasing in `tags.py`, but it doesn't need to strictly be lowercase, lowercase/shorter strings/single-words are more likely to be accidentally assigned when its actually referring to something else.

Reply:
I'm suggesting you remove any checks for casing by treating upper and lower case the same. Eg. if you check just for lower case, you don't need to look for `pits` and `Pits`. If you can point out major issues with this then fair enough

Reply:
That would be ideal yeah. There's certainly some pros to the way it is right now, but with the size the tags lists have gotten to recently, it probably warrants a change nowadays. We can amend this in another PR as you mentioned in discord.